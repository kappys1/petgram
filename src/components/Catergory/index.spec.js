// Category.test.js
/* global describe, it, expect */
import React from 'react'
// import renderer from 'react-test-renderer'
import { render } from '@testing-library/react'
import { Category } from '.'

describe('Category', () => {
  it('snapshot', () => {
    // const component = renderer.create(
    //   <Category />
    // )
    // const tree = component.toJSON()
    // expect(tree).toMatchSnapshot()

    const { asFragment } = render(<Category />)
    expect(asFragment()).toMatchSnapshot()
  })
})
