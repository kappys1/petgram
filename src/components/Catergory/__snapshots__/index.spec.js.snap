// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Category snapshot 1`] = `
<DocumentFragment>
  <a
    class="sc-bdfBQB TyDmP"
    href="/#"
  >
    <img
      alt="category image ?"
      class="sc-gsTEea cqupgU"
      src="https://i.imgur.com/dJa0Hpl.jpeg"
    />
    ?
  </a>
</DocumentFragment>
`;
